### PR TITLE
Add call for contributing on the front page

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

**Comment by ericholscher** [comment]
- [ ] Update footer to link to team page

File: readthedocs/templates/base.html [hunk 1]
```diff
@@ -85,17 +85,13 @@
 
         {% block footer-content %}
         <p>
-        Copyright {% getCopyrightYears %}.
-
-        {% blocktrans with "<a href='http://ericholscher.com/'>Eric Holscher</a>" as eric and "<a href='http://charlesleifer.com/'>Charles Leifer<a>" as charles and "<a href='http://bobbygrace.info/'>Bobby Grace</a>" as bobby and "<a href='http://djangodash.com/'>Django Dash</a>" as djangodash %}
-          Created by {{ eric }}, {{ charles }}, and {{ bobby }} for the 2010 {{ djangodash }}.
-        {% endblocktrans %}
+          Copyright {% getCopyrightYears %} Read the Docs, Inc
```
Comment (agjohnson) [inline readthedocs/templates/base.html]: Ugh copyright law. Not even sure if this makes sense. A couple issues:

* RTD Inc formation was a couple years into the project, this displays the start year
* AFAIK, copyright date is not necessary at all here
* We have multiple authors, not sure if this should instead read "respective owners"?
* Is a copyright even necessary?

File: readthedocs/templates/base.html [hunk 2]
```diff
@@ -85,17 +85,13 @@
 
         {% block footer-content %}
         <p>
-        Copyright {% getCopyrightYears %}.
-
-        {% blocktrans with "<a href='http://ericholscher.com/'>Eric Holscher</a>" as eric and "<a href='http://charlesleifer.com/'>Charles Leifer<a>" as charles and "<a href='http://bobbygrace.info/'>Bobby Grace</a>" as bobby and "<a href='http://djangodash.com/'>Django Dash</a>" as djangodash %}
-          Created by {{ eric }}, {{ charles }}, and {{ bobby }} for the 2010 {{ djangodash }}.
-        {% endblocktrans %}
```
Comment (agjohnson) [inline readthedocs/templates/base.html]: In adding the footer link to our contributor page, think it's important to not highlight single contributors on every page as well.

File: readthedocs/templates/base.html (hunk 1 above)
Comment (ericholscher) [inline readthedocs/templates/base.html]: Yea, we shouldn't have this be copyright the corporation. We can probably just remove it.

File: readthedocs/templates/base.html (hunk 2 above)
Comment (ericholscher) [inline readthedocs/templates/base.html]: Aye -- this should link to the team page in the docs, and/or the GH contributors page.

File: readthedocs/templates/homepage.html [hunk 3]
```diff
@@ -131,7 +148,7 @@
             <div class="module-wrapper">
 
               <div class="module-header">
-                <h3>{% trans "Recently Bookmarked Docs" %} <span class="small_head"><a href="{% url "bookmarks_list" %}">{% trans "(All Bookmarks)" %}</a></span></h3>
+                <h3>{% trans "Recently bookmarked docs" %} <span class="small_head"><a href="{% url "bookmarks_list" %}">{% trans "(All Bookmarks)" %}</a></span></h3>
```
Comment (ericholscher) [inline readthedocs/templates/homepage.html]: This should just be killed.

**Comment by ericholscher** [comment]
Think this should be ready for merging now.

**Comment by agjohnson** [comment]
LGTM

**Pull request merged**